git.md

```diff
@@ -87,6 +106,42 @@ The **footer** should contain any information about breaking changes and is also
 
 Breaking changes should start with the word `BREAKING CHANGE`: with a space or two newlines. The rest of the commit message is then used for this.
 
+
+
+### Pre-commit hook
+
+Commits are usually checked in the project CI, but you can configure [gitlint](https://jorisroovers.com/gitlint/latest/commit_hooks/#pre-commit) as a plugin for [pre-commit](https://pre-commit.com/#installation) hook to validate each commits locally.
+
+Simply install `pre-commit` using `pip`:
+```
+pip install pre-commit
+```
+
+Create a `.pre-commit-config.yaml` file at the root of the project with the following configuration:
+```
+repos:
+  - repo: https://github.com/jorisroovers/gitlint
+    rev: v0.19.1
+    hooks:
+      - id: gitlint
+        additional_dependencies: ["requests", "./gitlint-core[trusted-deps]"]
+
+```
+
+You then need to install the pre-commit hook like so: 
+```
+pre-commit install --hook-type commit-msg
+```
+
+`gitlint` should now be runned locally for each commits you make.
+
+If you want to manually trigger `gitlint` for your last commit message, use the following command:
+```
+pre-commit run gitlint --hook-stage commit-msg --commit-msg-filename .git/COMMIT_EDITMSG
+```
```

Comment:
Not sure it is relevant to recommend installing this `pre-commit` dependency. I would prefer documenting `gitlint` integration in a standard `pre-commit` hook (see [this reference](https://www.atlassian.com/git/tutorials/git-hooks#local-hooks) for example).

Your documentation may state that [Gitlint](https://jorisroovers.com/gitlint/latest/) should be installed globally or executed using Docker in a local shell script hook (`.git/hooks/pre-commit`), _e.g._:

```shell
#!/usr/bin/env bash

docker run --ulimit nofile=1024 -v $(pwd):/repo jorisroovers/gitlint
```

Reply:
Didn't bother executing with Docker, do you think I should?

Reply:
You are right. This is the way!

Reply:
I realize that my suggestion of a `pre-commit` hook was silly :sweat_smile: